docs/README.md

```diff
@@ -18,4 +18,12 @@ $ bundle exec jekyll serve
 # The site will now be reachable at http://127.0.0.1:4000/faraday/
 ```
 
+On newer Ruby versions (>= 3.0) `eventmachine` needs a little help to find OpenSSL 1.1 to get compiled (see <https://github.com/eventmachine/eventmachine/issues/936>). If you're using homebrew on macOS, you can do this:
+
+```bash
+brew install openssl@1.1
+bundle config build.eventmachine --with-openssl-dir=$(brew --prefix openssl@1.1)
+bundle install
+```
+
```

Comment:
Thank you for pointing this out @tisba.
Faraday doesn't bundle an `eventmachine` adapter since v2.0, so people should not have this issue unless they're using https://github.com/lostisland/faraday-em_http or https://github.com/lostisland/faraday-em_synchrony.

I'd suggest moving this suggestion in those gems because `eventmachine` is not a dependency in this gem

Reply:
What @olleolleolle said. This is about getting the docs running locally.

Reply:
really?! Well that's a surprise 😄!
In that case, thanks for the addition 🙏 !